core/src/plugins/filed/python/mariabackup/bareos-fd-mariabackup.py

```diff
@@ -211,6 +293,8 @@ def start_backup_job(self):
                     100,
                     "Import of module MySQLdb failed. Using command pipe instead\n",
                 )
+            # TODO Fix as this doesn't use the mysqldefault passed file and always try to connect
```

Comment:
This is still the case: it is an old problem since long time, the python mysql code path will try to connect to the default socket instead of the one described in .mysqldefault file.
This is not usually detected by end-user because the FD run as root on the DB host and usually this one is configured to backup a main default mysql/mariadb host.
It become visible if you try to run the systemtest as build user and run a "user" instance mysql.
It hasn't been detected before by our systemtest due to two things:
- we had a default mysql instance running before when we used several db and the code was trying to check the incremental from that instance.
- we often don't have the python mysql module installed, and then the systemtest used the external command